ui/desktop/electron-app/src/cli/path.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
+const { isWindows } = require('../helpers/platform.js');
+
+module.exports = {
+  // Returns boundary cli path
+  path: () => {
+    let name = 'boundary';
```

Comment:
How about `const name = (isWindows)? 'boundary.exe' : 'boundary';` ? not a big fan of one line conditionals, but in this case is pretty short.

Reply:
Good idea. Updated.